sys-apps/likwid/files/likwid-4.3.1-Makefile.patch

```diff
@@ -0,0 +1,21 @@
+--- likwid-likwid-4.3.1/Makefile.org	2018-02-26 13:59:28.421504363 +0800
```

Comment:
Please stop duplicating identical files. While at it, please prepend a commit removing existing duplicate files and updating 4.1.2 to use the files from 4.1.1.

Reply:
Agree that  likwid-4.1.2-Makefile.patch likwid-4.1.1-Makefile.patch are identical, but likwid-4.3.1-Makefile.patch patches different lines in the Makefile. The patch contents is the same but where they occur inside the Makefile has changed. Hope that makes sense
How do you suggest I proceed here?

Reply:
If it still applies, use the old patch.

Reply:
Old patch will not apply. Thus new patch needed.
I also after this version becomes stable want to drop all the older versions.
Simply not needed and will well with the patch file mess.